src/model.rs

```diff
@@ -54,6 +54,27 @@ impl Collections {
     /// Merge the `Collections` parameter into the current `Collections` by consecutively merging
     /// each collections representing the model.  Fails in case of id collision.
     pub fn merge(&mut self, c: Collections) -> Result<()> {
+        let Collections {
+            contributors: _,
+            datasets: _,
+            networks: _,
+            commercial_modes: _,
+            lines: _,
+            routes: _,
+            vehicle_journeys: _,
+            physical_modes: _,
+            stop_areas: _,
+            stop_points: _,
+            feed_infos: _,
+            calendars: _,
+            companies: _,
+            comments: _,
+            equipments: _,
+            transfers: _,
+            trip_properties: _,
+            geometries: _,
+            admin_stations: _,
+        } = c;
```

Comment:
No, the idea is to destructure `c` into variables, so no `: _`. It will warning on unused variables. Then you use these variables (removing `c.`). If you don't match all the field, it will not compile, if you don't use all the fields, it will warn

Reply:
That's the hole point, after the let, you don't use `c` anymore, you use `calendars` that is defined by the let.

Reply:
aaaaaaaaaaah ok